### PR TITLE
Add DisableEmbeddedSubtitles support in editor

### PR DESCRIPTION
**Pull request by matthew-jones-uk**
Adds the option to disable embedded subs for libraries

**Changes**
- Add the checkbox to enable embedded subs.
- Add the logic to get the checkbox status and pass to library options object.
- Add the logic to only display checkbox with shows, movies, music videos, and mixed content.
- Add en-gb and en-us translations.

The pull request to add this to the server is [here](https://github.com/jellyfin/jellyfin/pull/7054).

### CODE REVIEW

**Comment by crobibero** [comment]
Looks like the string files have been reformatted

**Comment by matthew-jones-uk** [comment]
> Looks like the string files have been reformatted

Oops, fixed.

File: src/components/libraryoptionseditor/libraryoptionseditor.js [hunk 1]
```diff
@@ -411,6 +411,12 @@ import template from './libraryoptionseditor.template.html';
             parent.querySelector('.chkEnableEmbeddedEpisodeInfosContainer').classList.add('hide');
         }
 
+        if (contentType !== 'tvshows' || contentType !== 'movies' || contentType !== 'musicvideos' || contentType !== 'mixed') {
```
Comment (crobibero) [inline src/components/libraryoptionseditor/libraryoptionseditor.js:414]:
```suggestion
        if (contentType === 'tvshows' || contentType === 'movies' || contentType === 'musicvideos' || contentType === 'mixed') {
```

I think these were reversed?
(checkbox shows for `book`)

Comment (matthew-jones-uk) [reply src/components/libraryoptionseditor/libraryoptionseditor.js]: Oops yeah, I think it's reversed.

**Review by crobibero — APPROVED** [review]
Confirmed working with the server PR

**Comment by matthew-jones-uk** [comment]
Just a heads-up here, this is going to be changed from a checkbox to a drop down with more options as per the discussion in the [server pull request](https://github.com/jellyfin/jellyfin/pull/7054).

**Comment by nielsvanvelzen** [comment]
Can you fix the merge conflicts?

**Pull request merged**